libs/shared/l10n/src/lib/branding.ftl

```diff
@@ -54,6 +54,14 @@
 -brand-google = Google
 -brand-paypal = PayPal
 -brand-name-stripe = Stripe
+-brand-amex = American Express
+-brand-diners = Diners Club
+-brand-discover = Discover
+-brand-jcb = JCB
+-brand-mastercard = Mastercard
+-brand-unionpay = UnionPay
+-brand-visa = Visa
+-brand-unbranded = Unbranded
```

Comment:
Is "Unbranded" ironically a brand, or should this be translated. We'd need a comment  for this either way.

Reply:
Would a comment like `# Generic payment card` be helpful?